backend/tilavarauspalvelu/tasks.py

```diff
@@ -185,18 +223,28 @@ def send_application_handled_email_task() -> None:
     EmailService.send_seasonal_booking_application_round_handled_emails()
 
 
-@app.task(name="update_expired_orders")
-@deprecated("Use 'refresh_expired_payments_in_verkkokauppa_task' instead. Can be removed in next release.")
-def update_expired_orders_task() -> None:
-    refresh_expired_payments_in_verkkokauppa_task()
-
-
-@app.task(name="refresh_expired_payments_in_verkkokauppa")
+@app.task(
+    name="refresh_expired_payments_in_verkkokauppa",
+    auto_create_name="Merkitse maksamattomat verkkokaupan tilaukset rauenneiksi",
+    auto_create_description=(
```

Comment:
Need to account for the post-handling payments; the description could be (somewhat more succinctly expressed than in Confluence):
`Merkitsee rauenneiksi maksutapahtumat (PaymentOrder-tietomalli), joiden määräaika on ohitettu.`

Reply:
Will change. All of the descriptions were taken directly from Production server descriptions